ghost/core/core/server/services/settings-helpers/SettingsHelpers.js

```diff
@@ -220,16 +222,46 @@ class SettingsHelpers {
     }
 
     /**
-     * Social web (ActivityPub) is enabled if:
-     * - Social web is enabled in the settings
-     * - 'ActivityPub' flag is enabled in the labs, for compatibility with Ghost 5.x
-     * - Config allows it (TODO)
-     * - Billing allows it (TODO)
+     * Calculated setting for Social web (ActivityPub)
      *
      * @returns {boolean}
      */
     isSocialWebEnabled() {
-        return this.settingsCache.get('social_web') === true && this.labs.isSet('ActivityPub');
+        // UI setting
+        if (this.settingsCache.get('social_web') !== true) {
+            logging.warn('Social web is disabled in settings');
+            return false;
+        }
+
+        // Labs setting
+        if (!this.labs.isSet('ActivityPub')) {
+            logging.warn('Social web is disabled in labs');
+            return false;
+        }
+
+        // Ghost (Pro) limits
+        if (this.limitService.isDisabled('limitSocialWeb')) {
+            logging.warn('Social web is not available for Ghost (Pro) sites without a custom domain, or hosted on a subdirectory');
+            return false;
+        }
+
+        // Social web (ActivityPub) currently does not support Ghost sites hosted on a subdirectory, e.g. https://example.com/blog/
+        const subdirectory = this.urlUtils.getSubdir();
+        if (subdirectory) {
```

Comment:
We currently have two sites with a subdirectory that have ActivityPub enabled in labs and are on the grandfathered list for that reason. Those two sites won't have the limit applied because they are on that grandfathered list. 

I wonder if we 
- Have to disable the setting for those sites
- Remove them from the grandfather list
- Communicate the reason why we disabled it with them

Probably something to discuss in Slack.

Reply:
Discussed in Slack, resolving